x-pack/plugin/logsdb/src/yamlRestTest/resources/rest-api-spec/test/40_source_mode_setting.yml

```diff
@@ -575,7 +547,7 @@ create an index with time_series index mode and disabled source:
                 time_series_dimension: true
 
   - match: { error.type: "mapper_parsing_exception" }
-  - match: { error.reason: "Failed to parse mapping: _source can not be disabled in index using [time_series] index mode" }
+  - match: { error.reason: "Failed to parse mapping: _source cannot be disabled in index using [time_series] index mode" }
```

Comment:
This is necessary because the error message introduced by the setting uses `cannot` instead of `can not`